### PR TITLE
Add photo to zenefits integration

### PR DESCRIPTION
**Pull request by cgoetze11**
1. Add photo to custom list items when available
2. Allow for multiple guide and custom list id combinations.

### CODE REVIEW

File: add_new_employee_webhook_receiver.py
Comment (ryeakle) [inline add_new_employee_webhook_receiver.py:22]: Would it make sense to have `fetch_ssm_params()` return a dictionary of form `{"{ssm param}": value, ...}` rather than a tuple here?

File: update_existing_employee_webhook_receiver.py
Comment (ryeakle) [inline update_existing_employee_webhook_receiver.py:49]: this should be fine as `photo_available = img_response.status_code == 200`

File: update_existing_employee_webhook_receiver.py
Comment (ryeakle) [inline update_existing_employee_webhook_receiver.py]: would it make sense to break out this `'/tmp/image.jpg'` into its own constant? Even though it's unlikely to change it seems like it's being referenced frequently

File: data_loader.py
Comment (ryeakle) [inline data_loader.py]: I like the direction this is going. Since this is always expecting a builder session, do you think it would make sense to make a super-light builder client class? Something like:
```python
class BuilderClient(object):
    """
    Helper class for making requests to the Builder API which handles things like raising exceptions on invalid 
    responses etc.
    """
    def __init__(self):
        self.builder_session = requests.Session()
        self.builder_session.headers.update({"Authorization": f"JWT {settings.builder_api_key}"})
    def post(...):
        ...
    def get(...):
        ....
```

**Review by ryeakle — COMMENTED** [review]
A few questions/comments for now, will take a look again later before this gets merged

File: data_loader.py
Comment (cgoetze11) [reply data_loader.py]: I like this! added it here https://github.com/Guidebook/guidebook-zenefits-integration/pull/2/commits/2cb7052176eca46bdf6a2ee5100339f81fa3f73c

File: add_new_employee_webhook_receiver.py
Comment (cgoetze11) [reply add_new_employee_webhook_receiver.py:22]: is a dictionary preferred? they both make sense to me

Comment (ryeakle) [reply add_new_employee_webhook_receiver.py:22]: Thinking about this more, if we make any changes here I think it should actually be a named tuple, not a dictionary. This would look something like:
```python
from collections import NamedTuple
...
def fetch_ssm_params():
    # ...
    SSMParams = NamedTuple('SSMParams', ['api_key', 'guide_and_list_ids', 'zenefits_app_key'])
    ...
    return SSMParams(api_key= , guide_and_list_ids= , ...)

# whenever we're referencing this:
ssm_params = fetch_ssm_params()

ssm_params.api_key ...
```
This might be overkill for our use case here though, so I'll leave it up to you on whether this is an improvement or not. I think at best it means simpler code changes if the return values of `fetch_ssm_params()` change. At worst, it's added complexity to something which might not change in the future. Generally though I think named tuples are a good tool to consider for situations like this.

**Review by andrewdieken — COMMENTED** [review]
shouldn't the `CustomlistDataBuilder` class be named `CustomlistItemDataBuilder`? It's not constructing custom list data it's constructing custom list item data for an already created custom list

**Comment by andrewdieken** [comment]
I know this is outside the scope of the work for this PR but it might be a nice improvement 🤷‍♂️

File: builder_client.py
Comment (andrewdieken) [inline builder_client.py:38]: how come you're not providing the option to raise an error here?

File: builder_client.py
Comment (andrewdieken) [inline builder_client.py]: do you not want to raise an error for `patch` requests?

File: README.md
Comment (andrewdieken) [inline README.md]: remove

File: add_new_employee_webhook_receiver.py
Comment (andrewdieken) [reply add_new_employee_webhook_receiver.py:22]: I'm indifferent about either way. There are few enough values returned now where I think it's manageable to keep as is. If more values are added in the future it for sure should be updated to a dict/named tuple

File: builder_client.py
Comment (cgoetze11) [reply builder_client.py:38]: I only provided it as optional for `posts` because there was a specific instance that we wanted to do something different https://github.com/Guidebook/guidebook-zenefits-integration/blob/main/add_new_employee_webhook_receiver.py#L48.

File: add_new_employee_webhook_receiver.py
Comment (cgoetze11) [reply add_new_employee_webhook_receiver.py:22]: Yeah, I think I'll leave it out on this one, as there are only three values. I added it for the google drive integration though, which has a lot more.

**Review by andrewdieken — APPROVED** [review]

**Pull request merged**
